layouts/partials/head.html

```diff
@@ -104,15 +104,13 @@
     {{ end }}
 
     <script async defer src="https://buttons.github.io/buttons.js"></script>
-    {{- if eq hugo.Environment "production" }}
```

Comment:
I removed this logic and added my own site key for the site preview. This will make it easier for people to provide feedback. I will change this back to the production set up when we are ready to merge this in.